### ISSUE TITLE
typescript bindings broken for 'noImplicitAny' mode

### ISSUE THREAD

**Issue by jefbarn** [ansman/validate.js#168]
Error:(3, 9) TS7006: Parameter 'attributes' implicitly has an 'any' type.
Error:(3, 21) TS7006: Parameter 'constraints' implicitly has an 'any' type.

should be
```
export declare interface ValidateJS {
  (attributes: any, constraints: any, options?: any): any;
  async(attributes: any, constraints: any, options?: any): Promise<any>;
  single(value: any, constraints: any, options?: any): any;
}
```

**Comment by jefbarn** [comment]
Pull request #169

**Comment by ansman** [comment]
Thanks, fixed in 0.11.1
